tests/utils/testrunner.go

```diff
@@ -0,0 +1,73 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package utils
```

Comment:
As test infra is evolving,  is testrunner seem to be right candidate to go under "utils" package or it go some test-infra  package, thoughts ?

Reply:
My understanding is that the definition of `test-infra` is the actual testing cluster where the e2e tests are running.

TestRunner is more like framework or test utilities working with go testing framework. So I would keep the package name `utils`. or maybe `framework` ? which one do you prefer ?

Reply:
I would move everything under `/utils` to the root, meaning one level up

Reply:
@yaron2  If we have everything under dapr/utils, these codes will be linked to Dapr runtime binaries :) I believe you do not want these test util codes to be the part of Dapr runtime binaries.

Reply:
I said the exact opposite.. take everything OUT of utils, meaning get rid of utils.

Reply:
And move everything there to the root of/tests.

Reply:
@yaron2 oh no... I do not think it is good decision to move all to `dapr/tests/`. TestRunner needs to be part of util or framework. To me. having separate package makes more sense. BTW, Why do you think we move them to dapr/tests ?

Reply:
Usually it's better to keep a flat structure if possible, also if we think TestRunner is a main entry point for running tests, it would be better to keep it under the root as it makes it more obvious that the runner is central.

Reply:
@yaron2 test runner is a main entry and central only for go tests in dapr/tests/e2e, but I designed and implemented test runner as a framework and util. /dapr/tests will includes only test start script, dapr_test.mk (makefile), and README.md in the next pr. for these reasons, I do not move these testrunner into /dapr/tests/.

Reply:
@yaron2 let me correct test runner is not a main entry and central for e2e test. it is designed as a framework. so the actual e2e test is running by below command. 

So, each go test in ./tests/e2e/* is the test main entry point. 

```
go test -v ./tests/e2e/... -tags=e2e
```

Reply:
Then I suggest util turns to runner, or testrunner.

Reply:
runner looks better.